force-app/main/default/lwc/platformDevCertCalculator/platformDevCertCalculator.js

```diff
@@ -0,0 +1,91 @@
+import { LightningElement, track } from 'lwc';
+const devFundWeight = 0.23;
+const processAutoWeight = 0.3;
+const userInterfaceWeight = 0.25;
+const testDebugWeight = 0.22;
+
+export default class PlatformDevCertCalculator extends LightningElement {
+
+    devFundamentals = 50;
+    processAutomation = 50;
+    userInterface = 50;
+    testDebugDeploy = 50;
+
+    certificationScore = 90;
+    numberOfQuestionsCorrect = 60;
+
+    showResources = false;
+
+    currentHistoryId = 0;
+
+    @track attemptHistory = [
+        {Id: 1, Score:50},
+        {Id: 2, Score:25}
+    ];
+
+    calculateScore(){
```

Comment:
Try adding a validation when if the values are negative or over 100% for the score and pop a toast that there was an error.